docs/whats-new/preview-6.md

```diff
@@ -0,0 +1,189 @@
+
+:::image type="content" source="media/preview-6/console-token.png" lightbox="media/preview-6/console-token.png" alt-text=".NET Aspire console with login token highlighted":::
+
+For more information, see [Explore the .NET Aspire Dashboard: Authentication](../fundamentals/dashboard/explore.md#dashboard-authentication).
+
+### New resources and components
+
+The following are the new resources and components added in this release:
+
+**Aspire.Hosting.Qdrant**: Provides a resource definition for a .NET Aspire AppHost to configure a Qdrant vector database resource.
+**Aspire.Qdrant.Client**: Provides a client library for interacting with a Qdrant vector database.
+
+From within an app host project, you can add a Qdrant resource like this:
+
+```csharp
+var qdrant = builder.AddQdrant("qdrant");
+
+var myService = builder.AddProject<Projects.MyService>()
+                       .WithReference(qdrant);
+```
+
+The component consuming project can the access the Qdrant client as shown in the following code:
+
+```csharp
+builder.AddQdrantClient("qdrant");
+```
+
+## App host updates
+
+In .NET Aspire preview 6, the app host was updated to include several new APIs and features.
+
+### Container runtime arguments
+
+You can now specify custom arguments to the docker or podman run command when running a container. This can be useful for accessing specifics features of the container runtime not exposed by aspire directly.
+
+```csharp
+builder.AddContainer("ollama", "ollama/ollama")
+       .WithVolume("/root/.ollama")
+       .WithBindMount("./ollamaconfig", "/root/")
+       .WithHttpEndpoint(port: 11434, name: "ollama")
+       .WithEntrypoint("/root/entrypoint.sh")
+       .WithContainerRunArgs("--gpus=all");
+```
+
+The preceding example uses the `WithContainerRunArgs` method to pass the `--gpus=all` argument to the container runtime. This is useful when running a container that requires access to a GPU.
+
+> [!NOTE]
+> .NET Aspire's orchestrator doesn't validate the arguments you pass to the container runtime. It's up to you to ensure that the arguments are valid and safe. These arguments can also conflict with the arguments that .NET Aspire uses to run the container.
+
+### Arguments on project
+
+Outside of using launch profiles, it's now possible to programmatically pass arguments to a project resources:
+
+```csharp
+builder.AddProject<Projects.Api>("api")
+       .WithArgs("name", "Aspire");
+```
+
+### Set a custom environment variable name for a connection string
+
+Resources in .NET Aspire can expose connection strings by implement `IResourceWithConnectionString`. This is used in conjunction with the `WithReference` method to pass the connection string to a project. This sets the fixed name `ConnectionStrings__{ResourceName}`. This can make it hard to integrate with systems that already have an existing convention of naming configuration keys. You can now set a custom environment variable name for the connection string:
```

Comment:
"This ..." was used in three sentences back-to-back, so I altered them a bit to make it flow better.

```suggestion
Resources in .NET Aspire can expose connection strings by implementing `IResourceWithConnectionString`. This interface is used in conjunction with the `WithReference` method to pass the connection string to a project. This sets the fixed name `ConnectionStrings__{ResourceName}`, which might make it hard to integrate with systems that already have an existing convention of naming configuration keys. To help with this problem, you can now set a custom environment variable name for the connection string:
```